deploy/charts/trust-manager/README.md

```diff
@@ -31,6 +31,7 @@ Kubernetes: `>= 1.25.0-0`
 | app.metrics.service.enabled | bool | `true` | Create a Service resource to expose metrics endpoint. |
 | app.metrics.service.servicemonitor | object | `{"enabled":false,"interval":"10s","labels":{},"prometheusInstance":"default","scrapeTimeout":"5s"}` | ServiceMonitor resource for this Service. |
 | app.metrics.service.type | string | `"ClusterIP"` | Service type to expose metrics. |
+| app.pkcs12Password | string | `nil` | Password for JKS and PKCS12 stores protection |
```

Comment:
I think these docs are a bit misleading. The default should be `""` (empty string) creating a password-less PKC12 truststore.

Reply:
> I think these docs are a bit misleading. The default should be `""` (empty string) creating a password-less PKC12 truststore.

I guess it makes code more clear and avoid of copying bundle data between function/method. 
The bundle data is tightly coupled to its encoder; imagine you would like to add `decode()`, `transform()` methods in the future, then it leads that the bundle data should be returned from one encoder method and immediately consumed by another one.

Reply:
I agree, it will be changed